bowler/query.py

```diff
@@ -895,13 +895,23 @@ class Fixer(BaseFix):
             PATTERN = pattern  # type: ignore
             BM_compatible = bm_compat
 
-            def transform(self, node: Node, capture: Capture) -> None:
+            def transform(self, node: Node, capture: Capture) -> Optional[Node]:
```

Comment:
Just realized the type signature should probably use the `LN` type for both `node` and the optional return value. Can you fix that while you're at it?